test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalPatchTests.cs

```diff
@@ -63,16 +74,23 @@ public async Task GivenAResource_WhenPatchingConditionallyWithOneMatch_TheServer
             using FhirResponse<Patient> response = await _client.CreateAsync(patient);
             Assert.Equal(HttpStatusCode.Created, response.StatusCode);
 
-            string patchDocument =
-           "[{\"op\":\"replace\",\"path\":\"/gender\",\"value\":\"female\"}, {\"op\":\"remove\",\"path\":\"/address\"}]";
+            using FhirResponse<Patient> jsonPatchResponse = await _client.ConditionalJsonPatchAsync<Patient>(
+                "Patient",
+                $"identifier={identifier}",
+                _patchDocumentJson);
 
-            using FhirResponse<Patient> patchResponse = await _client.ConditionalPatchAsync<Patient>(
+            using FhirResponse<Patient> fhirPatchResponse = await _client.ConditionalFhirPatchAsync<Patient>(
                 "Patient",
                 $"identifier={identifier}",
-                patchDocument);
-            Assert.Equal(AdministrativeGender.Female, patchResponse.Resource.Gender);
-            Assert.Empty(patchResponse.Resource.Address);
-            Assert.Equal(HttpStatusCode.OK, patchResponse.StatusCode);
+                _fhirPatchRequest);
+
+            Assert.Equal(AdministrativeGender.Female, jsonPatchResponse.Resource.Gender);
+            Assert.Empty(jsonPatchResponse.Resource.Address);
+            Assert.Equal(HttpStatusCode.OK, jsonPatchResponse.StatusCode);
+
+            Assert.Equal(AdministrativeGender.Female, fhirPatchResponse.Resource.Gender);
```

Comment:
For this test the JSON and FHIR patches should be in separate tests. Since they are working on the same patient if the JSON one succeeds then there are no unique changes made by the FHIR patch to check that it worked properly.

Reply:
Agreed. Was trying to be cute to remove duplication of code but it didn't really get us much. I separated these out.